docs/source/index.rst

```diff
@@ -26,15 +35,14 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    packages
 
 .. toctree::
+   :glob:
    :maxdepth: 1
    :caption: Python API
 
    torch
-   nn
+   nn/index.rst
    nn.functional
    tensors
-   tensor_attributes
```

Comment:
tensors, tensor_attributes, and tensor_view are a logical grouping: why did we separate them?

Reply:
For background, I moved tensor_attributes down because the displayed title is "Tensor Attributes". 
![image](https://user-images.githubusercontent.com/8042156/76778344-e5ec1180-6766-11ea-9cc9-263e220c3635.png)
The idea was to mostly alphabetize the docs (with torch, torch.nn still towards the top) and group bigger topics (e.g. Quantization, Distributed RPC) that deviate from the regular naming schema (e.g. torch.something). 

I like this idea -> "I think a more logical grouping would be: torch.Tensor is a general heading, and under torch.Tensor we have Tensor Attributes and Tensor Views." I'll make a new PR with this update.